config/locales/en.yml

```diff
@@ -4823,6 +4823,7 @@ See the %{link} to find out more about %{sitename}'s features and to start using
         invoice_attached_text: Please find attached an invoice for your recent order from
     user_mailer:
       reset_password_instructions:
+        dear_customer: "Dear customer,"
```

Comment:
I thought this sounded slightly strange. But then I saw we have the same in `shipped_email`, so there's a precedent 👍